Clinical-Genomics/scout
# Move current igv tracks to constants and let users choose which IGV tracks to display

fix #1872.

This PR does 4 things:
- Moves the IGV track info from alignviewers/controllers.py to the scout/constants (makes sense to me since they're all constants)
- Refactors the alignviewers.views so that most of its code is now in the controllers
- Introduces the possibility for users to customize which tracks they want to see from the tracks common to all cases (Genes, ClinVar and ClinVar CNV)
- Introduces tests for the alignviewers, which was largely untested before.

Note that **STRs variants page doesn't change** and the possibility to customize tracks is present only on variants and SV-variants page:
![image](https://user-images.githubusercontent.com/28093618/93733889-5e7cd080-fbd7-11ea-9703-11ffeeea245d.png)


**How to test**:
1. Install the branch on stage and go to a recently uploaded case containing alignment files.
1. Make sure that alignments of variants/cancer variants/ SV variants and STRs still work.
1. Update your users settings to visualize custom igv tracks and make sure the right tracks show up in igv.js browser

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
